### PR TITLE
fix: issue causing HtmlOverlayTool to misplace elements when 3D viewer was scaled on large screens

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: viewer/core/src/public/migration/Cognite3DViewer.ts [hunk 1]
```diff
@@ -978,16 +978,18 @@ export class Cognite3DViewer {
    */
   worldToScreen(point: THREE.Vector3, normalize?: boolean): THREE.Vector2 | null {
     this.camera.updateMatrixWorld();
-    const p = from3DPositionToRelativeViewportCoordinates(this.camera, point);
+    const p = new THREE.Vector3();
```
Comment (christjt) [inline viewer/core/src/public/migration/Cognite3DViewer.ts:981]: rename p to something more descriptive

File: examples/src/pages/Migration.tsx [hunk 2]
```diff
@@ -485,6 +486,12 @@ export function Migration() {
             {
               const { treeIndex, point, model } = intersection;
               console.log(`Clicked node with treeIndex ${treeIndex} at`, point);
+              const overlayHtml = document.createElement('div');
+              overlayHtml.innerText = `Node ${treeIndex}`;
+              overlayHtml.style.cssText = 'background: white; position: absolute;';
+              overlayTool.add(overlayHtml, point);
+              console.log(overlayHtml);
```
Comment (christjt) [inline examples/src/pages/Migration.tsx:493]: we are starting to get a lot of `console.log` clutter in the migration example. Kinda annoying when you are trying to debug something. Maybe we can remove some?

File: viewer/core/src/utilities/worldToViewport.ts [hunk 3]
```diff
@@ -4,32 +4,76 @@
...
-  relativePosition.addScalar(1).multiplyScalar(0.5); // Is it better to have a multiply with a 0.5, -0.5 vector?
-  return relativePosition;
+  const { renderSize, position } = worldToViewportVars;
+  const canvas = renderer.domElement;
+  const pixelRatio = 1; // renderer.getPixelRatio();
+  renderer.getSize(renderSize);
+
+  // map to normalized device coordinate (NDC) space
+  position.copy(position3D);
+  position.project(camera);
```
Comment (christjt) [inline viewer/core/src/utilities/worldToViewport.ts:37]: I guess the project method does perspective divide?

**Review by christjt — CHANGES_REQUESTED** [review]

File: viewer/core/src/utilities/worldToViewport.ts (hunk 3 above)
Comment (larsmoa) [reply viewer/core/src/utilities/worldToViewport.ts:37]: It does, ref https://github.com/mrdoob/three.js/blob/e62b253081438c030d6af1ee3c3346a89124f277/src/math/Vector3.js#L301 + https://github.com/mrdoob/three.js/blob/e62b253081438c030d6af1ee3c3346a89124f277/src/math/Vector3.js#L262

File: examples/src/pages/Migration.tsx (hunk 2 above)
Comment (larsmoa) [reply examples/src/pages/Migration.tsx:493]: Oops, removed

**Review by christjt — APPROVED** [review]

**Pull request merged**